VERSION

```diff
@@ -1 +1 @@
-0.1.10
+0.1.11
```

Comment:
it should be `0.2.0` as this library follow semantic versioning and this add functionality